<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriveButton</class>
 <widget class="QPushButton" name="DriveButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>186</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>72</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="drive_icon">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>11</y>
     <width>56</width>
     <height>58</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>56</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res/app.qrc">:/icons/app.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="drive_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string notr="true">Local Volume(C:)</string>
   </property>
  </widget>
  <widget class="QLabel" name="drive_mountpoint">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>256</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string notr="true">/mnt/XXX</string>
   </property>
  </widget>
  <widget class="QLabel" name="win_icon">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res/app.qrc">:/icons/win.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res/app.qrc"/>
 </resources>
 <connections/>
</ui>
